Automattic/vip-go-mu-plugins
# Update/jp 10

## Description

We've upgraded Jetpack 10.0 branch from the beta version to its release one.


## Changelog Description

### Plugin Updated: Jetpack 10.0

Making Jetpack 10.0 release available.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Set Jetpack version manually to 10.0 - ` define( ‘VIP_JETPACK_PINNED_VERSION’, ‘10.0’ );`
3. Test Jetpack functionality
